app/src/main/res/values/colors.xml

```diff
@@ -6,4 +6,5 @@
     <color name="titleTextColor">#bb0414</color>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mataku/SunsetScrob/blob/358f4d3d5d1451906a8d4af2536fd5c14fbe83c0/app/src/main/res/values/colors.xml#L6"></span>The resource <code>R.color.titleTextColor</code> appears to be unused</td>
    </tr>
  </tbody>
</table>